solr/conf/solrconfig.xml

```diff
@@ -143,6 +143,12 @@
          subject_addl_tsim^10
        </str>
 
+       <!-- for advanced search number text box -->
+       <str name="qf_number">
```

Comment:
If we will keep the naming for others as `*_qf`, what do you think changing this to `number_qf`. Or we can change the others as `qf_*`?

Reply:
Do we need to add a `pf_number` for boosting?

Reply:
The short answer is "I don't know." I did think about it though and my thoughts were that we are only looking at two fields for this search. Two numbers-only fields. So what good would a phrase field be? I would assume there aren't even any spaces in these fields. ?